test/integration/makerV2Manager_invest.js

```diff
@@ -0,0 +1,217 @@
+// How to run this test file:
```

Comment:
why do you have separated some tests in /integration?

Reply:
These are the previous Kovan tests. We could just delete them but since I may have cut some corners when deploying and setting up MCD in my ganache tests, there may be some value to keep them around as an additional set of tests that touch a real world MCD deployment. Open to both.

Reply:
Yes, let's keep them.